memcached/memcached.php

```diff
@@ -683,6 +683,7 @@ class Memcached
     public const GET_PRESERVE_ORDER = 1;
 
     /**
+     * (PECL memcached &gt;= 3.0.0)<br/>
      * A flag for <b>Memcached::get()</b>, <b>Memcached::getMulti()</b> and
      * <b>Memcached::getMultiByKey()</b> to ensure that the CAS token values are returned as well.
      * @link https://php.net/manual/en/memcached.constants.php
```

Comment:
I've gave that form a thought, and decided in favor of the first solution:
the `HAVE_MSGPACK` const documentation is the sole example of `@since` stanza in this file.
I'm not against it...

Do we need to double state the version?
```
     * Available as of Memcached 3.0.0.
     * @since 3.0.0
```